docs/dev/framework/caching.md

```diff
@@ -230,13 +230,28 @@ When working with DCAs in the Contao back end you don't have to register callbac
 tags are being invalidated. This is because Contao invalidates a certain set of tags whenever a back end entry is created,
 updated or deleted. The tags are as follows:
 
-* `contao.db.<table-name>`
 * `contao.db.<table-name>.<id>`
-* `contao.db.<parent-table-name>` (only if there is a parent table defined)
+* `contao.db.<table-name>` OR `contao.db.<parent-table-name>` (depending on whether a parent table is defined or not)
 * `contao.db.<parent-table-name>.<pid>` (only if there is a parent record)
+* `contao.db.<child-table-name>.<cid>` (only if there is a child record)
+
+Imagine you  edited a news article with ID 42. Contao will now automatically send an invalidation request to the 
+reverse proxy to invalidate all responses associated with the following tags: 
+
+* `contao.db.tl_news.42`
+* `contao.db.tl_news_archive`
+* `contao.db.tl_news_archive.1`
+* `contao.db.tl_content.420`
```

Comment:
```suggestion
* `contao.db.tl_content.420`
* `contao.db.tl_content.421`
```
I would add another content element here to make it clear that there can be multiple of them.

Reply:
Updated accordingly